### PR TITLE
 OpenAPI Spec - Aditya

### PR DESCRIPTION
**Pull request by adiSuper94**

### CODE REVIEW

File: reference/wazzup-open-api-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+              schema:
+                $ref: '#/components/schemas/User'
+              examples:
+                Get User Alice Smith:
+                  value:
+                    id: 142
+                    firstName: Alice
+                    lastName: Smith
+                    email: alice.smith@gmail.com
+                    dateOfBirth: '1997-10-31'
```
Comment (AdishRao) [inline reference/wazzup-open-api-spec.yaml:33]: We are not taking the users date of birth

File: reference/wazzup-open-api-spec.yaml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: unique ID of the message
+          minLength: 1
+          readOnly: true
+        text:
+          type: string
+          maxLength: 300
```
Comment (AdishRao) [inline reference/wazzup-open-api-spec.yaml:264]: min length so user doesn't send blank message?

**Comment by AdishRao** [comment]
@ashwin2303 @shashankbhat10 Check out from line 208 onwards, only user details has been made

**Comment by adiSuper94** [comment]
> @ashwin2303 @shashankbhat10 Check out from line 208 onwards, only user details has been made

and Message details

**Comment by AdishRao** [comment]
Line 208 onwards looks good

**Comment by AdishRao** [comment]
line 208 onwards looks good

**Comment by ashwin2303** [comment]
@adiSuper94 and @AdishRao  I'm guessing you were referring to the line number based on lines in the 'Files Changed' section. Nevertheless, I have reviewed the open-api-spec.yaml file and the first overview looks good. We'll use this as a base and build the api. 

@shashankbhat10 and I will review and add/mention any changes if necessary.

**Pull request merged**